bin/node/cli/src/service.rs

```diff
@@ -243,6 +253,7 @@ macro_rules! new_full {
 				telemetry_on_connect: Some(service.telemetry_on_connect_stream()),
 				voting_rule: grandpa::VotingRulesBuilder::default().build(),
 				prometheus_registry: service.prometheus_registry(),
+				finality_subscribers: Some(finality_proof_subscription),
```

Comment:
If we pass the finality proof subscription manager via grandpa link this should not be necessary.